### PR TITLE
Added pitching guidelines and communication links

### PR DESCRIPTION
**Pull request by felixfaassen**
Added pitching guidelines and communication links to the for staff onboarding page so we provide new staff with starting points on how to explain the Foundation for Public Code and the work we do as well as point out our communication principles, activities and methods.

-----
[View rendered contributor-guides/for-staff.md](https://github.com/publiccodenet/about/blob/felixfaassen-onboard-communication/contributor-guides/for-staff.md)

### CODE REVIEW

**Comment by Ainali** [comment]
Should it also be linked from the communications index? There's also a bullet point mentioning pitching in events/attending-events.md that could be linked

**Comment by felixfaassen** [comment]
Build errors are cause by a bug in MD lint see my issue and fix here: #737

**Review by ElenaFdR — COMMENTED** [review]
Hey, I'm ready to approve this - these look like very useful changes. Given that there's a new version of MDL out, do you still want to force it to build from source?

**Comment by felixfaassen** [comment]
@ElenaFdR thanks for your suggestions I commit them to this branch

File: activities/events/attending-events.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ Before using this guide, make sure to [get a decision on attending the event](de
 * Consider using a shared document to do the planning in, perhaps using [this template](events-planning-template.md).
 * Plan your [travels](../staff-management/travel.md).
 * Prepare a team chat (that is capable of notifications) if you are more than one staff member attending.
-* Rehearse and make sure you attend to the event with a brief pitch explaining what our Foundation does. Tailor the pitch to the audience you are going to meet.
+* Rehearse and make sure you attend to the event with a brief [pitch explaining what the Foundation for Public Code does](../communication/pitching.md). Tailor the pitch to the audience you are going to meet.
```
Comment (ElenaFdR) [inline activities/events/attending-events.md:26]:
```suggestion
* Make sure you attend the event with a brief, well-rehearsed [pitch explaining what the Foundation for Public Code does](../communication/pitching.md). Tailor the pitch to the audience you are going to meet.
```

File: contributor-guides/for-staff.md [hunk 2]
```diff
@@ -7,6 +7,8 @@ explains: The basics for being a staff member of the Foundation for Public Code
 
 Our staff are expected to follow and uphold both the [Community code of conduct](../CODE_OF_CONDUCT.md) and the [Staff code of conduct](../organization/staff-code-of-conduct.md). Make sure you are familiar with both.
 
+In your work you will need to introduce and explain the Foundation for Public Code. You'll answer questions about why we exist and what we do. The [pitching guidance](../activities/communication/pitching.md) provides a good starting point to learn how to briefly explain our work to others. We also provide a guide which describes our [communications activities](../activities/communication/index.md) and how we communicate with eachother.
```
Comment (ElenaFdR) [inline contributor-guides/for-staff.md:10]:
```suggestion
In your work you will need to introduce and explain the Foundation for Public Code. You'll answer questions about why we exist and what we do. The [pitching guidance](../activities/communication/pitching.md) provides a good starting point to learn how to briefly explain our work to others. We also provide a guide which describes our [communications activities](../activities/communication/index.md) and how we communicate with each other.
```

**Review by ElenaFdR — APPROVED** [review]
Totes ready to go, and a nice addition to website navigability!

**Comment by ElenaFdR** [comment]
This closes #735

**Pull request merged**